### ISSUE TITLE
Hopefully simple one: Don't scramble original filenames

### ISSUE THREAD

**Issue by JustACuteGirlLookingForAManToFixHerBugs** [beat-machine/beat-machine#50]
Hey! Is it possible to keep original filename when downloading? Right now it comes out as a random string of numbers and letters. If you're concerned about duplicates - the string can be appended to the original name.

**Comment by branchpanic** [comment]
Yes, I'll transfer this to the webapp repository since the CLI lets you specify the output filename.
